pr_testing/test-cmssw-provides.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -xe
+if [ $# -ne 6 ]; then
+  echo Usage: test-cmssw-provides.sh scram_arch pkgtools_branch cmsdist_tag build_dir week_num cmssw_release
+  exit 1
+fi
+
+SCRAM_ARCH=$1
+PKG_TOOL_BRANCH=$2
+CMSDIST_TAG=$3
+BUILD_DIR=$4
+WEEK_NUM=$5
+CMSSW_RELEASE=$6
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"  # Absolute path to script
+CMS_BOT_DIR=$(dirname ${SCRIPTPATH})  # To get CMS_BOT dir path
+
+rm -rf $WORKSPACE/test-provides
+mkdir -p $WORKSPACE/test-provides
+cd $WORKSPACE/test-provides
+if [ ! -d $WORKSPACE/pkgtools ]; then
+  git clone git@github.com:cms-sw/pkgtools --depth 1 -b $PKG_TOOL_BRANCH
+else
+  ln -s $WORKSPACE/pkgtools .
+fi
+
+if [ ! -d $WORKSPACE/cmsdist ]; then
+  git clone git@github.com:cms-sw/cmsdist --depth 1 -b $CMSDIST_TAG
+else
+  ln -s $WORKSPACE/cmsdist .
+fi
+
+RPM_CMD="/cvmfs/cms-ib.cern.ch/sw/$(uname -m)/week${WEEK_NUM}/common/cmspkg -a $SCRAM_ARCH env -- rpm"
+
+if [ ! -d ${WORKSPACE}/$BUILD_DIR/$SCRAM_ARCH/var/lib/rpm ]; then
+  if [ ! -z ${CMSSW_FULL_RELEASE_BASE} ]; then
+    CMSSW_RELEASE_PATCH=$CMSSW_RELEASE
+    CMSSW_RELEASE=$(echo ${CMSSW_FULL_RELEASE_BASE} | rev | cut -d '/' -f 1 | rev)
```

Comment:
`$(basename ${CMSSW_FULL_RELEASE_BASE})` should return the same information :-)

Reply:
also looks like we are doing `if [ ! -d ${WORKSPACE}/$BUILD_DIR/$SCRAM_ARCH/var/lib/rpm ]; then` check twice ( once here and other one https://github.com/cms-sw/cms-bot/pull/1936/files#diff-51dd032b1bff11f296c5943e95c62abe50c617f7a5e23511fd0d94c096b4feb9R65 ). May be you can set `RPM_DB_PATH="${WORKSPACE}/$BUILD_DIR/$SCRAM_ARCH/var/lib/rpm"` before this block and set to `RPM_DB_PATH="/cvmfs/cms-ib.cern.ch/sw/`uname -m`/week${WEEK_NUM}/${SCRAM_ARCH}/var/lib/rpm"` instead . and then at https://github.com/cms-sw/cms-bot/pull/1936/files#diff-51dd032b1bff11f296c5943e95c62abe50c617f7a5e23511fd0d94c096b4feb9R64 just do `cp -r ${RPM_DB_PATH} build/${SCRAM_ARCH}/var/lib/`

Reply:
`RPM_CMD` is only used inside this block ... right ? then better to move it inside the block